doc/api/vm.md

```diff
@@ -38,6 +38,10 @@ The options when creating a script are:
   script's methods.
 - `timeout`: a number of milliseconds to execute `code` before terminating
   execution. If execution is terminated, an [`Error`][] will be thrown.
+- `breakEvalOnSigint`: if `true`, the execution will be terminated when
+  `SIGINT` (Ctrl+C) is received. This will disable existing handlers for the
+  event that have been attached via `process.on("SIGINT")` permanently.
```

Comment:
One can emit it as an event using `process.emit`, sure, but right now the default action for `SIGINT` would be reverted to aborting the process.

I’ll see if it’s possible to do the underlying code changes here using the uv_signal stuff… it seems multiple uv_loops can install event handlers for the same event, so I’ll give it a try. That would allow `process.on("SIGINT", …)` handlers to work normally (I guess).